### PR TITLE
feat(configuration): receipt_printing

### PR DESCRIPTION
**Pull request by alghanor**
add fine grained salesperson style and conditions

https://tillhub.atlassian.net/browse/TM-6597

### CODE REVIEW

File: lib/v1/configurations/items/receipts.js [hunk 1]
```diff
@@ -37,23 +83,8 @@ const cartReceipt = {
   additionalProperties: false,
   properties: {
     ...commonReceipt,
-    print_salesman_policy: oneOf({
```
Comment (alghanor) [inline lib/v1/configurations/items/receipts.js:40]: covered by `style`

File: lib/v1/configurations/items/receipts.js [hunk 2]
```diff
@@ -37,23 +83,8 @@ const cartReceipt = {
...
-      description: 'Defines if and how the salesperson will be printed on the receipt',
-      type: 'string',
-      enum: [
-        'number',
-        'name',
-        'none'
-      ]
-    }),
-    print_salesperson_policy: oneOf({
```
Comment (alghanor) [inline lib/v1/configurations/items/receipts.js:49]: covered by `conditions`

**Comment by eljefedelrodeodeljefe** [comment]
@alghanor mergeable

**Pull request merged**